source/making_payments/index.html.md.erb

```diff
@@ -87,37 +122,96 @@ HTTP with test accounts.
 
 The URL must be no longer than 2,000 characters, and must not be JSON-encoded because backslashes are invalid characters in the API.
 
-Refer to the <a
-href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
-target="blank">GOV.UK Pay API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more
-information.
-
 ## Receiving the API response
 
-The response to the ‘Create new payment’ API call will include the URL for the
-payment. You can get this from either:
+Example response:
+
+```json
+{
+  "created_date": "2020-03-03T16:17:19.554Z",
+  "state": {
+    "status": "created",
+    "finished": false,
+  },
+  "_links": {
+    "self": {
+      "href": "https://publicapi.payments.service.gov.uk/v1/payments/hu20sqlact5260q2nanm0q8u93",
+      "method": "GET"
+   },
+    "next_url": {
+      "href": "https://publicapi.payments.service.gov.uk/secure/bb0a272c-8eaf-468d-b3xf-ae5e000d2231",
+      "method": "GET"
+    },
+    ...
+  },
+  "amount": 14500,
+  "reference" : "12345",
+  "description": "Pay your council tax",
+  "return_url": "https://your.service.gov.uk/completed",
+  "payment_id": "hu20sqlact5260q2nanm0q8u93",
+  "payment_provider": "worldpay",
+  "provider_id": "10987654321",
+  ...
+}
+```
+
+The response to the ‘Create new payment’ API call will include the URL for the payment. You can get this from either:
 
 * the `Location` response header
 * the `self` section of `_links` in the JSON body
 
-The URL contains the GOV.UK Pay `paymentId`, which is automatically
-generated and identifies the payment. A GET request to this URL
-will return information about the payment and its status.
+The URL contains the GOV.UK Pay `paymentId`, which is automatically generated and identifies the payment. A GET request to this URL will return information about the payment and its status.
```

Comment:
What's the reason for referring to the `paymentId` in camelCase here rather than as `payment_id`? This may be a standard convention, but just curious.

Reply:
Hmm good question. This text predates me, but it looks like we have a convention of using `paymentId` for the URL parameter, and `payment_id` for the API field.

I think it's ok to leave as is for now (as it's been this way a while). But it is a bit odd, so I'll make a note to revisit this later.